### ISSUE TITLE
Make e2e tests to run from branch

### ISSUE THREAD

**Issue by igoychev** [podkrepi-bg/frontend#1351]
### Which area(s) of Podkrepi.bg are affected? (leave empty if unsure)

TypeScript

### Describe the Bug

Currently the e2e tests for a new branch are run from a newly built master instead from the new branch which contains the new code.

### To Reproduce

1. Make a deliberate error in code so that the tests fail on localhost
2. Publish it as a new branch
3. See how tests will pass when "run tests" label is added

### Expected Behavior

We need the tests on a branch to run from the branch itself, because there might be new tests adapted for the new code.

**Comment by dimitur2204** [comment]
I am not sure that this is the case, because I was recently making new e2e tests on #1336 and this run for example has 24 tests (https://github.com/podkrepi-bg/frontend/actions/runs/4157716833/jobs/7192414709).
This is the latest run from master (https://github.com/podkrepi-bg/frontend/actions/runs/4203735250/jobs/7293506065) and it has 23.

**Comment by igoychev** [comment]
We saw the problem with these PRs:
After changing the english translations of donation and fees here: https://github.com/podkrepi-bg/frontend/pull/1331 the tests passed, but then when merged in master, the tests started to fail.
So I had to fix them in a separate PR: https://github.com/podkrepi-bg/frontend/pull/1345 and where although fixed the tests were failing in the branch and I had to force the merge to master. Then in the master they passed well.
